src/Migrations/M058ReplaceTagsRedirections.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+use Red_Item;
+
+/**
+ * Retire Tags redirection option in favor of the redirection tool.
+ */
+class M058ReplaceTagsRedirections extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     * phpcs:disable Squiz.PHP.DiscouragedFunctions
+     */
+    protected static function execute(MigrationRecord $record): void
+    {
+        if (!is_plugin_active('redirection/redirection.php')) {
+            return;
+        }
+
+        $redirections = self::get_existing_redirections();
+
+        if (!empty($redirections)) {
+            $group_id = self::create_new_group();
+            $status = self::add_redirections($group_id, $redirections);
+        } else {
+            $status = "No redirections were found.";
+        }
+
+        $record->add_log(print_r($status, true));
+        print_r($status);
+    }
+    // phpcs:enable SlevomatCodingStandard.Functions.UnusedParameter
+    // phpcs:enable Squiz.PHP.DiscouragedFunctions
+
+    /**
+     * Gets the existing tag redirections and returns them.
+     * Also, removes the the existing tag redirections to avoid conflicts with the Redirections tool.
+     */
+    private static function get_existing_redirections(): array
+    {
+        $redirections = [];
+
+        $terms = get_terms(array(
```

Comment:
For consistency in array declarations, could you please use `[]` instead of `array()`?